transcendence-app/seeds/001_seed_users.ts

```diff
@@ -0,0 +1,69 @@
+import { Knex } from 'knex';
+import { faker } from '@faker-js/faker';
+import { rmSync } from 'fs';
+import { Password } from '../src/shared/password';
+import { configService, createRandomAvatar, defaultUsernames } from './utils';
+import { join } from 'path';
+import { AVATARS_PATH } from '../src/shared/constants';
+
+const USERS_NUMBER = 5000;
+
+const createRandomUser = (avatarId: string) => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const username = faker.helpers.unique(faker.internet.userName, [
+    firstName,
+    lastName,
+  ]);
+  return {
+    username: username.slice(-20),
+    email: faker.internet.email(firstName, lastName).slice(-50),
+    fullName: faker.name.fullName({ firstName, lastName }),
+    avatarId,
+  };
+};
+
+const createUserWithPassword = async (username: string, avatarId: string) => {
+  const hashedPassword = await Password.toHash(username);
+  return {
+    username,
+    email: `${username}@${username}.com`,
+    fullName: username,
+    avatarId: avatarId,
+    password: hashedPassword,
+  };
+};
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries and local files
```

Comment:
If we delete the users, we also delete the chatrooms because of the `ON DELETE CASCADE` on the `ownerId` column.

So we can also delete all the avatar files here.